src/_help/hubs/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
+
+With hubs, your API ecosystem is displayed on a single page, where documentation can be sorted by categories.
+
+Hubs also provide a unified changelog, making it easy to identify all changes across all the documentation it contains at a glance. Search is also global, allowing you to search across all documentation.
+
+## Create a hub
+
+In your dashboard, click on Hubs then on Create a hub.
+
+![](/images/help/create-hub.png)
+
+You will be asked to choose a name for the hub, which will be visible to everyone. The slug is the portion of the URL that you can customize. You can also choose to use a custom domain instead. Finally, you can decide whether this hub is visible to everyone or only to members of your organization.
+
+![](/images/help/hub-creation.png)
+
+## Add documentation to a hub
+
+### Using the webapp
+#### Deploy a new documentation
+From the "Hubs" tab, click on "Create Documentation" to add it to the desired hub. You can then specify the documentation's name, its slug, and its access level (Public or private).
+#### Transfer an existing documentation
+From the "Standalone Documentation" tab, select the relevant documentation and go to its settings. The "Hubs Settings" section allows you to choose which hub to associate this documentation with. You can also specify a category if necessary.
+
+![](/images/help/move-standalone-doc.png)
+### Using the CLI
```

Comment:
This section isn't obvious to me. 
My expectation in a paragraph under "Add documentation to a hub" > "Using the CLI" is to get straightforward info about how to deploy a new doc, or transfer an existing doc, to a hub. Like written in "using the webapp", just above.
Also, I'm guessing we don't support yet the doc transfer from one hub to another with the CLI, but we should then mention it clearly, so that people don't keep looking for the info.
Finally, I would also make the paragraph very actionable, so that no browsing to another doc page is necessary (especially as users will also have to go and fetch their tokens in another page). Even if it's great to point to additional resources on the CLI for other use cases than deploying a doc to a hub.

My content recommendation here:
"The CLI allows you to deploy new documentation directly to a hub. It does not support yet the possibility to create a hub, nor to transfer a doc from one hub to another (you will need to do that via the webapp, as described above).

To deploy a new doc to a hub, execute the following command:
```
bump deploy your/doc.yml --doc my-doc --hub hub-slug-or-id --token my-doc-token
```

[Find here the commands and more information about the CLI.](/help/bump-cli)

Reply:
👍